MODULE.bazel

```diff
@@ -6,15 +6,11 @@ module(
     compatibility_level = 1,
 )
 
-# Lower-bound (minimum) versions of direct dependencies
+# Lower-bounds (minimum) versions for direct runtime dependencies
+bazel_dep(name = "aspect_bazel_lib", version = "2.7.1")
```

Comment:
Should we try leaving some lower and just do rules_js + rules_nodejs? This was 2.5 in the `archive_override` can we keep that?

Reply:
Minimum has to be bumped to 2.7.1 to pickup the latest fixes for tar extract and for compat